measurement_octree_filename: /home/titoirfan/etc/soca_octomap_ws/src/soca_octomap/soca_octomap/example/octomaps/meas_octomap.ot
ground_truth_octree_filename: /home/titoirfan/etc/soca_octomap_ws/src/soca_octomap/soca_octomap/example/octomaps/ref_octomap.bt
dose_threshold: 100.0

save_difference_map: true
difference_map_save_path: /home/titoirfan/etc/soca_octomap_ws/src/soca_octomap/soca_octomap/example/octomaps/diff_octomap.bt

x_min: -2.5
x_max: 2.5
y_min: -2.5
y_max: 2.5
z_min: -0.5
z_max: 1.5